### ISSUE TITLE
Installation of 4.2.2 produces a Download Error

### ISSUE THREAD

**Issue by charjl** [neuropoly/spinalcordtoolbox#3180]
### Description

Installation of v. 4.2.2 of SCT, or shimming-toolbox which requires it, causes a download error. Prior to the download failure, the output indicates ```Unexpected: link doesn't provide a filename``` for https://osf.io/u79sr/?pid=6zbyf/?action=download
as well as  https://www.neuro.polymtl.ca/_media/downloads/sct/20191029_PAM50.zip
A user brought this issue up on the SCT forum (https://forum.spinalcordmri.org/t/sct-version-4-2-2-download-problems/607)

### Steps to Reproduce

0. Run the install script to Install SCT 4.2.2

1. Observe install fails

**Expected behavior:** Console completes without errors and product is downloaded and installed.

**Actual behavior:** Console shows a download error, and the installation fails
<details>
 <summary>Console Output for 4.2.2 Download Failure</summary>
```
Spinal Cord Toolbox (4.2.2)

Folder /home/char/sct_4.2.2/bin has been created.

Trying URL: https://osf.io/bt58d/?action=download
Downloading 20190930_sct_binaries_linux.tar.gz...
Status: 100%|##############################| 81.1M/81.1M [00:42<00:00, 1.89MB/s]

Create temporary folder (/tmp/sct-20210118132924.184006-0psdvi1d)...

Unzip data to: /tmp/sct-20210118132924.184006-0psdvi1d

Check if folder already exists on the destination path...

Remove temporary folders...
Done!


All requirements installed!


. /home/char/sct_4.2.2/python/bin/activate /home/char/sct_4.2.2/python


Installing data...


rm -rf /home/char/sct_4.2.2/data


mkdir -p /home/char/sct_4.2.2/data


cd /home/char/sct_4.2.2/data


sct_download_data -d PAM50


--
Spinal Cord Toolbox (4.2.2)


Trying URL: https://osf.io/u79sr/?pid=6zbyf/?action=download
Unexpected: link doesn't provide a filename

Trying URL: https://www.neuro.polymtl.ca/_media/downloads/sct/20191029_PAM50.zip
Unexpected: link doesn't provide a filename

Download error

Create temporary folder (/tmp/sct-20210118132933.492410-np30uzys)...

Unzip data to: /tmp/sct-20210118132933.492410-np30uzys
Traceback (most recent call last):
  File "/home/char/sct_4.2.2/scripts/sct_download_data.py", line 248, in <module>
    res = main()
  File "/home/char/sct_4.2.2/scripts/sct_download_data.py", line 140, in main
    unzip(tmp_file, dest_tmp_folder, verbose)
  File "/home/char/sct_4.2.2/scripts/sct_download_data.py", line 174, in unzip
    if compressed.endswith('zip'):
AttributeError: 'NoneType' object has no attribute 'endswith'

ERROR: Command failed.


Installation failed!

Please copy the historic of this Terminal (starting with the command install_sct) and paste it in a new created topic on SCT's forum:

--> http://forum.spinalcordmri.org/c/sct

```
</details>

**Comment by joshuacwnewton** [comment]
This turned out to be a duplicate of https://github.com/neuropoly/spinalcordtoolbox/issues/2588, and was fixed by #2671 for more recent versions of SCT. 

Thank you for reporting, though, @charjl! :sweat_smile:

**Comment by charjl** [comment]
Perfect :D
